docs/notebooks/low_level.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>nit: "since version 2 pandas", "Since v2.0, pandas"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-pandas/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>